### PR TITLE
Use qcthat

### PR DESCRIPTION
**Pull request by jonthegeek**
Add qcthat and copilot setup and auto-tag tests with issues.

### CODE REVIEW

File: tests/testthat/test-req_pkg_user_agent.R [hunk 1]
```diff
@@ -32,14 +32,14 @@ test_that("req_pkg_user_agent adds a package agent and url", {
...
+test_that("Corner case of no supplied name works. (#29)", {
   existing <- stringi::stri_rand_strings(1, 10)
   expect_identical(
     .lib_user_agent_append(existing, name = NULL, version = "x"),
     existing
   )
 })
 
-test_that("Corner case of no supplied agent works.", {
+test_that("Corner case of no supplied agent works. (#29)", {
```
[the remark this answers is not in the document]
Comment (jonthegeek) [reply tests/testthat/test-req_pkg_user_agent.R]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/api2r/nectar/pull/64#discussion_r2801202674)
